src/load_inputs/load_generators_data.jl

```diff
@@ -30,9 +30,11 @@ function load_generators_data!(setup::Dict, path::AbstractString, inputs_gen::Di
 	# Set of storage resources with symmetric charge/discharge capacity
 	inputs_gen["STOR_SYMMETRIC"] = gen_in[gen_in.STOR.==1,:R_ID]
 	# Set of storage resources with asymmetric (separte) charge/discharge capacity components
-	inputs_gen["STOR_ASYMMETRIC"] = gen_in[gen_in.STOR.==2,:R_ID]
+	STOR_ASYMMETRIC = gen_in[gen_in.STOR.==2,:R_ID]
```

Comment:
Why do you do this for STOR_ASYMMETRIC and STOR_ALL but not STOR_SYMMETRIC? Also this is separate from the purpose of this PR, correct? That's fine if so but wanted to check.